### DISCUSSION TITLE
Curved shock discrepancy for Roe and JST schemes?

### DISCUSSION THREAD

**Discussion by ChinmayU78** [su2code/SU2#1299] [Q&A] [answered]
Hi,
I'm currently simulating a sphere in a Mach 6 flow with an unstructured mesh, and I noticed that the bow shock is very distorted when I use the Roe scheme, however the results seem more correct with a smooth curved shock when using the JST scheme. I am confused as to why the Roe scheme does not seem to work in this situation, despite being an upwind convective scheme? The first image is the result with the Roe scheme, and the second one is with the JST scheme. 
![image](https://user-images.githubusercontent.com/74565109/120951589-d3188080-c70e-11eb-9766-e673b1f5fdcd.png)
![image](https://user-images.githubusercontent.com/74565109/120951599-d7449e00-c70e-11eb-9678-06fdef2fe247.png)
Thanks,
Chinmay

**Comment by erangit** [comment, marked as answer]
The Roe results are expected:  The Roe flux splitting scheme has known deficiencies dealing with strong detached SW (the famous carbuncle phenomena).  Roe suggested a carbuncle-free modification but it is not implemented in SU2 and it is not commonly accepted as successful.   It seems that the JST provided a smooth solution. In the wake, you can see that it is much more dissipative. I would have tried one of the AUSM, HLLC, or SLAU schemes for this kind of strong SW

**Reply by ChinmayU78** [reply]
Thank you for the response, that makes sense.
